app/validators/phone_number_validator.rb

```diff
@@ -0,0 +1,7 @@
+class PhoneNumberValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
+    unless value =~ /\A0|[1-9]\d{0,3}-\d{1,4}-\d{4}\z/
+      record.errors[attribute] << (options[:message] || "is not an phone number")
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Line is too long. [81/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)